lib/api/api.js

```diff
@@ -105,6 +106,7 @@ const api = {
     bucketGetCors,
     bucketGetVersioning,
     bucketGetWebsite,
+    bucketGetLocation,
```

Comment:
It has already been merged in Arsenal :)
I have a branch ready on 
- Integration: https://github.com/scality/Integration/pull/433
- Utapi: https://github.com/scality/utapi/pull/84
I have run an E2E 5 days ago with these branches and it passed :)